apm-agent-core/src/main/java/co/elastic/apm/agent/impl/transaction/TraceContext.java

```diff
@@ -382,6 +382,11 @@ public Span createSpan() {
         return tracer.startSpan(fromParent(), this);
     }
 
+    @Override
+    public Span createSpan(long epochMicros) {
+        return tracer.startSpan(fromParent(), this, epochMicros);
```

Comment:
Another one that escapes the parent reference counting?

Reply:
In this case, there is no parent span to count references for